src/sheldon_result.erl

```diff
@@ -25,19 +25,27 @@
         ]).
 
 -export_type([ result/0
+             , misspelled_word/0
+             , line_number/0
              ]).
 
--type result() :: ok | #{ result  => [string()]
-                        , bazinga => string()
-                        }.
+-type line_number() :: non_neg_integer().
+-type misspelled_word() ::
+  #{ line_number := non_neg_integer()
```

Comment:
yeap, I wrote that line_number() but I forgot to update the misspelled_word()...